bigquery/CHANGES.md

```diff
@@ -1,543 +1,525 @@
 ### 1.2.1 - 2021-XX-XX
 
+1. Update all dependencies to latest versions.
```

Comment:
Can we move all the cosmetic changes to a separate PR ?

Reply:
Done.